includes/functions/metadata.php

```diff
@@ -716,8 +716,8 @@ function resource_data_init() {
 
 	$archival_links->add_field(
 		[
-			'name'        => __( 'Wayback Machine Link', 'learning-commons-framework' ),
-			'description' => __( 'A link or links to an archival copy of the resource on the <a href="https://web.archive.org">Wayback Machine</a>. If the resource spans multiple pages on the Wayback Machine, you may add multiple links.', 'learning-commons-framework' ),
+			'name'        => __( 'The Internet Archive (archive.org)', 'learning-commons-framework' ),
```

Comment:
Maybe change to this for more consistency with Perma.cc field label?

```suggestion
			'name'        => __( 'Internet Archive (archive.org) link', 'learning-commons-framework' ),
```